cranelift/codegen/src/isa/s390x/lower/isle.rs

```diff
@@ -1060,7 +1060,6 @@ impl IsleContext<'_, '_, MInst, S390xBackend> {
 #[inline]
 fn lane_order_for_call_conv(call_conv: CallConv) -> LaneOrder {
     match call_conv {
-        CallConv::WasmtimeSystemV => LaneOrder::LittleEndian,
         CallConv::Tail => LaneOrder::LittleEndian,
```

Comment:
@uweigand should CallConv::Winch be LittleEndian too?

Reply:
We don't support Winch at all on s390x right now, so this probably doesn't matter at this point.  If and when we ever add support for Winch, we can re-think this, based on what the actual use case would be.